### PR TITLE
[7.0.x] JBPM-4646 - case management - case file data audit trail

### PR DESCRIPTION
**Pull request by mswiderski**
depends on https://github.com/kiegroup/jbpm/pull/812

### CODE REVIEW

File: kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/cases/CaseFileDataItem.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 201y Red Hat, Inc. and/or its affiliates.
```
Comment (psiroky) [inline kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/cases/CaseFileDataItem.java]: This looks weird, maybe some mis-configuration in the IDE template?

Comment (mswiderski) [inline kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/cases/CaseFileDataItem.java]: not really, it's just simple typo

Comment (psiroky) [inline kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/cases/CaseFileDataItem.java]: ah, ok :) I thought the `y` represents the last digit of current year, but it somehow wasn't replaced by the actual value.

Comment (mswiderski) [inline kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/cases/CaseFileDataItem.java]: fixed

**Pull request merged**